packages/ui/src/utils/setUserAgent/constants.ts

```diff
@@ -1,24 +1,66 @@
 import {
+  AuthUserAgentInput,
+  GeoUserAgentInput,
+  InAppMessagingUserAgentInput,
+  StorageUserAgentInput,
   setCustomUserAgent,
   SetCustomUserAgentInput,
   AuthAction,
+  GeoAction,
+  InAppMessagingAction,
+  StorageAction,
   Category,
 } from '@aws-amplify/core/internals/utils';
 
 // replace type once Category input types are available
 export const ACCOUNT_SETTINGS_INPUT_BASE: Omit<
-  SetCustomUserAgentInput,
+  AuthUserAgentInput,
   'additionalDetails'
 > = {
   apis: [AuthAction.DeleteUser, AuthAction.UpdatePassword],
   category: Category.Auth,
 };
 
-// replace type once Category input types are available
-export const IN_APP_MESSAGING_INPUT_BASE: Omit<
+export const AUTHENTICATOR_INPUT_BASE: Omit<
   SetCustomUserAgentInput,
   'additionalDetails'
 > = {
-  apis: undefined, // TODO
+  apis: [
+    AuthAction.SignUp,
+    AuthAction.ConfirmSignUp,
+    AuthAction.ResendSignUpCode,
+    AuthAction.SignIn,
+    AuthAction.FetchMFAPreference,
+    AuthAction.UpdateMFAPreference,
+    AuthAction.SetUpTOTP,
```

Comment:
Where do we use these?

Reply:
In the example apps? I just went off of those because they seems to demonstrate all the functionality of connected components. But you obv know more about the authenticator so if you can provide a list of the apis it uses I'll update